service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionControllerSubmittedTest.java

```diff
@@ -277,71 +267,59 @@ void shouldSendFailedPaymentNotificationOnHiddenDisplayAmountToPay() {
         });
     }
 
-    @Test
-    void shouldNotifyHmctsAdminWhenTheLocalAuthorityHasSubmittedAReturnedCase() {
-        CaseDetails caseDetails = enableSendToCtscOnCaseDetails(NO);
-        caseDetails.getData().put(DISPLAY_AMOUNT_TO_PAY, NO.getValue());
 
-        CallbackRequest callbackRequest = buildCallbackRequest(caseDetails, RETURNED);
-        postSubmittedEvent(callbackRequest);
+    @Nested
+    class CaseResubmission {
 
-        checkUntil(() -> verify(notificationClient).sendEmail(
-            eq(AMENDED_APPLICATION_RETURNED_TO_THE_ADMIN),
-            eq(HMCTS_ADMIN_EMAIL),
-            anyMap(),
-            eq(caseId.toString())));
-
-        checkThat(() -> verify(notificationClient, never()).sendEmail(
-            eq(AMENDED_APPLICATION_RETURNED_TO_THE_ADMIN),
-            eq(CTSC_EMAIL),
-            anyMap(),
-            eq(caseId.toString())));
-    }
+        final State state = RETURNED;
 
-    @Test
-    void shouldNotifyCtscAdminWhenTheLocalAuthorityHasSubmittedAReturnedCase() {
-        CaseDetails caseDetails = enableSendToCtscOnCaseDetails(YES);
-        caseDetails.getData().put(DISPLAY_AMOUNT_TO_PAY, NO.getValue());
+        @BeforeEach
+        void init() {
+            when(documentDownloadService.downloadDocument(any())).thenReturn(DOCUMENT_CONTENT);
+        }
 
-        CallbackRequest callbackRequest = buildCallbackRequest(caseDetails, RETURNED);
-        postSubmittedEvent(callbackRequest);
+        @Test
+        void shouldNotifyAdminAndCafcassWhenCaseIsResubmitted() {
+            CaseDetails caseDetails = enableSendToCtscOnCaseDetails(NO);
+            caseDetails.getData().put(DISPLAY_AMOUNT_TO_PAY, YES.getValue());
 
-        checkUntil(() ->
-            verify(notificationClient).sendEmail(
-                eq(AMENDED_APPLICATION_RETURNED_TO_THE_ADMIN),
-                eq(CTSC_EMAIL),
-                anyMap(),
-                eq(caseId.toString())));
+            CallbackRequest callbackRequest = buildCallbackRequest(caseDetails, state);
+            postSubmittedEvent(callbackRequest);
 
-        checkThat(() ->
-            verify(notificationClient, never()).sendEmail(
-                eq(AMENDED_APPLICATION_RETURNED_TO_THE_ADMIN),
-                eq(HMCTS_ADMIN_EMAIL),
-                anyMap(),
-                eq(caseId.toString())));
-    }
+            checkUntil(() -> resubmissionNotificationsSent(HMCTS_ADMIN_EMAIL));
+            checkThat(this::paymentNotTakenAndNoMoreEmailsSent);
+        }
 
-    @Test
-    void shouldNotNotifyAdminOfAReturnedCaseWhenTheCaseIsSubmittedFromOpenState() {
-        CaseDetails caseDetails = enableSendToCtscOnCaseDetails(YES);
-        caseDetails.getData().put(DISPLAY_AMOUNT_TO_PAY, NO.getValue());
+        @Test
+        void shouldNotifyCtscAndCafcassWhenCaseIsResubmitted() {
```

Comment:
Are we testing feature toggle value here? Not sure if much value testing but if the logic is in the controller we need. to live with it.